### DISCUSSION TITLE
User guide available?

### DISCUSSION THREAD

**Discussion by rdaaron** [kangyu-california/PersistentWindows#290] [Q&A]
Greetings. Love this product!

Is there a user's guide? It would seem not as far as looking around here. I tried searching for anyone doing a how-to or walkthrough to no avail. If you could point me to anything written, I'd consider running with that material and contribute back what I do. Let me know.

Thanks!
Rob

**Comment by kangyu-california** [comment]
There is no dedicated user guide beside the project home page.
If you want to try some advanced features, take a look at online help at https://github.com/kangyu-california/PersistentWindows/blob/master/Help.md
